<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select_lang</class>
 <widget class="QWidget" name="select_lang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select language</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Noto Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="lang_ok">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>92</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
